Program.cs

```diff
@@ -1,7 +1,48 @@
 ﻿namespace Training {
+
+   /// <summary>A program for calculating the number of chocolates that can be bought with a given inputs and the number of wrappers required for an extra chocolate.</summary>
    internal class Program {
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello, World!");
+
+      /// <summary>The entry point of the program.</summary>
+      static void Main () {
+         string xPrompt = "Enter the amount of money: ";
+         int money = GetInput (xPrompt);

```

Comment:
```suggestion
         int money = GetInput ("Enter the amount of money:  ");

```
Could have avoided unnecessary variable for prompt